rllib/offline/estimators/qreg_tf_model.py

```diff
@@ -0,0 +1,52 @@
+from ray.rllib.models.utils import get_initializer
+from ray.rllib.policy import Policy
+from ray.rllib.policy.sample_batch import SampleBatch
+from typing import Dict, List, Union
+
+from ray.rllib.models.catalog import ModelCatalog
+from ray.rllib.models.tf.tf_modelv2 import TFModelV2
+from ray.rllib.utils.framework import try_import_tf
+from ray.rllib.utils.typing import TensorType
+
+tf1, tf, tfv = try_import_tf()
+
+
+class QRegTFModel:
```

Comment:
actually does this need to be a ModelV2?

Reply:
No, I had made them ModelV2 for consistency earlier but have since removed it